AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/InitialSync/InitialSyncOrchestrator.swift

```diff
@@ -25,6 +28,8 @@ final class AWSInitialSyncOrchestrator: InitialSyncOrchestrator {
     typealias SyncOperationResult = Result<Void, DataStoreError>
     typealias SyncOperationResultHandler = (SyncOperationResult) -> Void
 
+    private var initialSyncOperationSinks: [String: AnyCancellable]
```

Comment:
Either protect this with a queue, or make it an AtomicDictionary

Reply:
updated